extension/agenthealth/handler/useragent/useragent.go

```diff
@@ -78,6 +79,15 @@ func (ua *userAgent) SetComponents(otelCfg *otelcol.Config, telegrafCfg *telegra
 		ua.outputs.Add(output.Config.Name)
 	}
 
+	// Adding SELinux status
+	if os.Getenv("SELINUX") == "true" {
+		ua.inputs.Add("SELinux/Enabled")
+		log.Printf("SELinux status: Enabled")
```

Comment:
How often does this get logged?

Reply:
Is there a way to make it only run once?